src/azure/azHttpClient.ts

```diff
@@ -242,7 +242,7 @@ export class AzHttpClient {
   }
 
   private getRoleAssignmentUrl(scope: string, roleAssignmentId: string): string {
-    return `https://management.azure.com${scope}/providers/Microsoft.Authorization/roleAssignments/${roleAssignmentId}?api-version=${this.roleApiVersion}`
+    return `${this.managementUrl}${scope}/providers/Microsoft.Authorization/roleAssignments/${roleAssignmentId}?api-version=${this.roleApiVersion}`
```

Comment:
Not sure whether all environments support the API versions being used for the different APIs. Good to test E2E!

Reply:
Will need to test it properly